### PR TITLE
Added support for OLO, Slovakia suource

### PR DESCRIPTION
**Pull request by mniejo**

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/olo_sk.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        "registrationNumber": "Registration number"
+    }
+}
+
+PARAM_TRANSLATIONS = {  # Optional dict to translate the arguments, will be shown in the GUI configuration form as placeholder text
+    "en": {
+        "street": "Enter street name",
+        "registrationNumber": "Enter registration number"
+    }
+}
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/olo_sk.py:50]: please remove `enter` from the string

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/olo_sk.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    "piatok": "Friday",
+    "sobota": "Saturday",
+    "nedeľa": "Sunday"
+}
+
+#### Arguments affecting the configuration GUI ####
+
+HOW_TO_GET_ARGUMENTS_DESCRIPTION = {  # Optional dictionary to describe how to get the arguments, will be shown in the GUI configuration form above the input fields, does not need to be translated in all languages
+    "en": "HOW TO GET ARGUMENTS DESCRIPTION"
+}
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/olo_sk.py:36]: Either fill in correct information or remove this Variable. This placeholder text would be shown in the configuration GUI at the moment

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/olo_sk.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        dates = dates_str.split(";")
+        dates = [date.strip() for date in dates]
+        dates = [date for date in dates if date]
+        return [datetime.datetime.strptime(date, "%d.%m.%Y").date() for date in dates]
+
+    def fetchWasteType(self, waste_type: str) -> str:
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+            "authorization": "Bearer ae84ae0982c2162a81eb253765ceaa8593abd9105c71954cf5c9620b0178cbb6"
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/olo_sk.py:71]: Is there another way to get this header, I don't like hard coding a Bearer token

File: doc/source/olo_sk.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# fcc Environment (VYLOŽ SMETI APP)
```
Comment (5ila5) [inline doc/source/olo_sk.md:1]: Headline copy-paste leftovers?

File: doc/source/olo_sk.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    - name: olo_sk
+      args:
+        street: STREET
+
+```
+
+### Configuration Variables
+
+**street**
+*(String) (required)*
```
Comment (5ila5) [inline doc/source/olo_sk.md:19]: registrationNumber missing

File: doc/source/olo_sk.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## Example
+
+```yaml
+waste_collection_schedule:
+    sources:
+    - name: olo_sk
+      args:
+        street: Jantarova 47
```
Comment (5ila5) [inline doc/source/olo_sk.md:28]: registrationNumber missing

File: doc/source/olo_sk.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+```
+
+```yaml
+waste_collection_schedule:
+    sources:
+    - name: olo_sk
+      args:
+        city: Jasovska 8
+```
```
Comment (5ila5) [inline doc/source/olo_sk.md:37]: city is not a valid argument

**Review by 5ila5 — CHANGES_REQUESTED** [review]

**Comment by mniejo** [comment]
Sorry. This PR was created by mistake on unfinished code. I'll be back with a new one in few days.

**Pull request closed without merging**